src/test/java/com/hocs/test/pages/markup/MarkUpDecision.java

```diff
@@ -116,7 +116,7 @@ public void selectSecondAnsweringUnitFromDropdown() {
     }
 
     public void selectSecondSignOffMinisterFromDropdown() {
-        signOffMinisterDropdown.selectByIndex(1);
+        signOffMinisterDropdown.selectByIndex(2);
```

Comment:
I'm guessing this has been changed because there is now a "Please Select" or something similar, which is index(0). If so, I would change the logic to how it was and rename the method to be ```selectFirstSignOffMininsterFromDropdown```. This logic is just about getting an option that not the default